spinalcordtoolbox/scripts/sct_deepseg.py

```diff
@@ -314,6 +320,29 @@ def main(argv: Sequence[str]):
         # Use the result of the current model as additional input of the next model
         fname_prior = fname_seg
 
+        # write JSON sidecar file
+        sidecar_json = {
+            'GeneratedBy': [
+                {
+                    'Name': 'spinalcordtoolbox',
+                    "Version": __version__,
```

Comment:
`Name` should be the "Pipeline used", so it was suggested to do:

```json
    "Name": "spinalcordtoolbox: sct_deepseg -i <> -task <> [...]",
```

Reply:
We may also want to consider scrubbing the full paths from the input for privacy reasons (e.g. by calling `basename`)